TF_Settings_Web/src/Components/Pages/Camera/QuickSetup/Calibration/CalibrationComponents.tsx

```diff
@@ -131,14 +129,12 @@ interface CalibrationCancelButtonProps {
 
 export const CalibrationCancelButton: React.FC<CalibrationCancelButtonProps> = ({ onCancel, buttonStyle }) => {
     return (
-        <IconTextButton
-            buttonStyle={{ ...setupButtonStyle, ...buttonStyle }}
-            icon={''}
-            alt=""
+        <TextButton
+            buttonStyle={{ ...cancelSetupButtonStyle, ...buttonStyle }}
             title="Cancel Setup"
-            titleStyle={setupButtonTitleStyle}
-            text={''}
-            textStyle={{ display: 'none' }}
+            titleStyle={{ display: 'none' }}
```

Comment:
Is this swapping of the text and title styles intentional?

Reply:
Yeah - using it as a title (H1) gave it some default styling I didn't want